docs/source/scheduling.rst

```diff
@@ -20,13 +19,13 @@ but with different performance characteristics.
 
 Dask has two families of task schedulers:
 
-1.  **Single machine scheduler**: This scheduler provides basic features on a
+1.  **Single machine scheduler**: this scheduler provides basic features on a
```

Comment:
Why remove the capitalization here?  This feels like the start of a sentence to me.  I think that the phrase "Single machine scheduler" is more of a label for this item than the start of the sentence.

Reply:
I've completely mixed up the conventions used here with another project I've been doing. Indeed, the capitalization should be used at the beginning. My confusion came from the Pandas documentation where they don't use capitalization in these cases (examples [here](https://pandas.pydata.org/pandas-docs/stable/basics.html#attributes-and-the-raw-ndarray-s), [here](https://pandas.pydata.org/pandas-docs/stable/basics.html#descriptive-statistics) and [here](https://pandas.pydata.org/pandas-docs/stable/basics.html#aligning-objects-with-each-other-with-align)).

I'll fix this immediately